src/pixbuf.c

```diff
@@ -13,18 +13,8 @@ GdkPixbuf *pixbuf_get_from_state(struct swappy_state *state) {
   return pixbuf;
 }
 
-void pixbuf_save_state_to_folder(GdkPixbuf *pixbuf, char *folder) {
+void write_file(GdkPixbuf *pixbuf, char *path) {
```

Comment:
I think this function should be `static`.

Reply:
Did you this?

Reply:
oh no, missed it